common/lib/mandrill_client/client.py

```diff
@@ -70,7 +71,46 @@ def send_mail(self, template_name, user_email, context, attachments=[], subject=
                 message=message,
             )
             log.info(result)
-        except mandrill.Error, e:
+        except mandrill.Error as e:
+            # Mandrill errors are thrown as exceptions
+            log.error('A mandrill error occurred: %s - %s' % (e.__class__, e))
+            raise
+        return result
+
+    def send_action_plan_mail(self, template_name, context, attachments=[], receiver_emails=[], sender_email=None,
```

Comment:
- Can we please `send_mail` generic enough to tackle all the requirements of action planner? I have gone through the previous commit as well, where you made changes in `send_email`, I guess @qasimgulzar suggested to move those sanities in a separate methods, instead of creating a completely new method which is duplicating the core logic of `send_email`.
- We can think it as like anyone can use images or reply to feature.
- Also after adding new params if method header gets too long, split it to multiple lines.

Reply:
yes i have  made that changes as we have decided above.